infra/gcb/build_and_run_coverage.py

```diff
@@ -32,6 +32,14 @@
 UPLOAD_REPORT_URL_FORMAT = 'gs://%s/{0}/reports/{1}' % COVERAGE_BUCKET_NAME
 
 
+def skip_build(message):
+  # Exit with 0 not to mark code coverage job as failed.
```

Comment:
Change # to """

Reply:
Also this comment needs fixing this is just not for coverage job failure.

Reply:
Done! This is for coverage job only (filename is `build_and_run_coverage.py`) :)